miniupnpd/netfilter_nft/nftnlrdr_misc.c

```diff
@@ -32,23 +32,45 @@
 #include <linux/ipv6.h>
 
 #include <libmnl/libmnl.h>
+#include <libnftnl/table.h>
+#include <libnftnl/chain.h>
 #include <libnftnl/rule.h>
 #include <libnftnl/expr.h>
 
+#include "../commonrdr.h"
 #include "nftnlrdr_misc.h"
 #include "../macros.h"
-#include "../upnpglobalvars.h"
+
+#define DEBUG 1
```

Comment:
that's an issue. You should build with` CFLAGS=-DDEBUG` or `#define DEBUG 1` in config.h if you want to enable debug

Reply:
Oops... sorry, that was meant to be a temporary thing, I just forgot to remove it. I've noticed I have a couple of debug messages I also meant to remove. I'll commit a fix.